### ISSUE TITLE
Clean up dlls after on systray.Quit()

### ISSUE THREAD

**Issue by luckcolors** [getlantern/systray#18]
Hello.
I wanted to ask if it would be possible to delte the dlls in appdata when you run systray.Quit()
i had this code in the systray.Quit function before but it didn't work because for some reason the files were still in use.

```
    err := os.RemoveAll(dllDir)
    if err != nil {
        log.Debugf("Unable to delete temporary dll directory %v: %v", err)
    }
```

**Comment by joesis** [comment]
Sorry @luckcolors I didn't notice the issue till now. It looks like an untypical requirement. Why do you want to delete it after each run?

Currently `systray` using `syscall.LazyDLL` to load the DLL file, which doesn't have any method to unload it. Per https://github.com/golang/go/wiki/WindowsDLLs, it can be changed to lower level `LoadLibrary` and `FreeLibrary`. PR is welcome.

**Comment by meskio** [comment]
Can this issue be closed now that #44 has being merged?

**Comment by luckcolors** [comment]
Yeah sure. 👍
